.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```

Comment:
You shouldn't have touched this config though, please change and use the config file provided by microverse and re-run for linting errors  
[ [see config here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) ]

Reply:
![Screen Shot 2020-04-12 at 1 19 39 PM](https://user-images.githubusercontent.com/40712459/79068551-b249cc00-7cbf-11ea-99de-11a7caef5389.png)